rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :book, :customer
+
+  def initialize(date, book, customer)
+    @date = date
+
+    @book = book
+    @book.book_rentals.push(self)
+
+    @customer = customer
+    @customer.rentals.push(self)
+  end
+end
```

Comment:
- [ ] Good job so far but I noticed that you have a `customer` ,  **from the project requirement a `rental` belongs to a `book` and also belongs to a `person`,** not a customer. Please fix it because as a developer it is very important to always follow the requirements because they establish a foundation for product vision, scope, cost, and schedule and they ultimately must target finished product quality and performance. :+1: